app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -17,42 +20,60 @@ import androidx.compose.ui.unit.dp
 import coil.compose.AsyncImage
 import coil.request.ImageRequest
 
-data class MovieCardViewState (
+
+data class MovieCardViewState(
     val imageUrl: String,
     val isFavorite: Boolean
-        )
+)
 
+@OptIn(ExperimentalMaterialApi::class)
 @Composable
 fun MovieCard(
     movieCardViewState: MovieCardViewState,
+    onCardClick: () -> Unit,
+    onLikeButtonClick: () -> Unit,
     modifier: Modifier = Modifier
-){
+) {
     Card(
         shape = RoundedCornerShape(percent = 10),
         elevation = 10.dp,
-        modifier = modifier.size(130.dp, 200.dp)
-    ){
-        Box{
+        onClick = onCardClick,
+        modifier = modifier
+    ) {
+        Box(modifier = Modifier) {
             AsyncImage(
-                model = ImageRequest.Builder(LocalContext.current).data(movieCardViewState.imageUrl).build(),
+                model = movieCardViewState.imageUrl,
                 contentDescription = movieCardViewState.imageUrl,
-                modifier = modifier
-                    .clickable {}
+                modifier
+                    .size(130.dp, 200.dp)
                     .fillMaxSize(),
                 contentScale = ContentScale.Crop
 
             )
+
             FavoriteButton(
                 isFavorite = movieCardViewState.isFavorite,
-            modifier.padding(start= 5.dp, top=5.dp).align((Alignment.TopStart)))
+                onClick = onLikeButtonClick,
+                modifier = modifier
+                    .padding(start = 5.dp, top = 5.dp)
+                    .clickable{onLikeButtonClick()}
```

Comment:
remove this as you passing `onLikeButtonClick` already through component parameter